ui/src/app/admin/@tab/manage-person/page.tsx

```diff
@@ -0,0 +1,20 @@
+import { managePersonResults } from '@/lib/fetchers';
+import PersonTable from '@/app/admin/_components/personTable';
+import { memberAttributeResults } from '@/lib/actions';
+
+const PersonTab = async (searchParams) => {
+    const searchUid = searchParams.searchParams.searchUid;
```

Comment:
Also I'm not sure why the searchParams objects would make you access .searchParams